content/en/cloud-computing.md

```diff
@@ -10,7 +10,9 @@ tags: ["infrastructure", "", ""]
 Cloud computing is a model that offers compute resources like CPU, network, and disk capabilities on-demand over the internet. 
 Cloud computing gives users the ability to access and use computing power in a remote physical location. 
 Cloud providers like AWS, GCP, Azure, DigitalOcean, and others all offer third parties 
-the ability to rent access to compute resources in multiple geographic locations. 
+the ability to rent access to compute resources in multiple geographic locations.
+With a layer of [abstraction](/abstraction/) built on top of one another, this architecture is the foundation 
```

Comment:
We are not speaking of Cloud Computing as an architecture, hence introducing architecture here might confuse people. Rather stick to model. Besides that, while what you add here might be true and I get what you mean, I don't believe that it really adds value to the definition of Cloud Computing. I guess that people who are not familiar with everything might be rather confused how abstractions built on top of one another can be a foundation for IaaS. PaaS and SaaS. TBH, I would not add the last sentence.

Reply:
+1 on keeping it simple. If we don't need it, let's drop it. We are trying to create some sort of minimal viable definition. What do people need to know to understand the concept? Once they get that, they can dig deeper with other resources.